templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>The part where you say "You can specify which type of cell cluster (SOM or meta) by setting cell_cluster_col accordingly. It should be set to 'cell_som_cluster' or 'cell_meta_cluster' is no longer needed (since we do both).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>